low-level/json_msgs/messages/base_msg.py

```diff
@@ -39,6 +41,21 @@ def __init__(self):
     def getJson(self):
         raise NotImplementedError("Subclasses should implement this!")
 
+    def add_common_resources(self, jsonMsg, request_type):
+        try:
+            if jsonMsg.get("message").get(
+                request_type).get("info") is None:
+                return
+            conf_list = ["site_id", "node_id", "rack_id", "cluster_id"]
+            conf = fetch_config(conf_list)
+            for key in conf_list:
+                if jsonMsg["message"][request_type]["info"].get("key") is None:

```

Comment:
key in quotes here? we need var from line 51 i believe !

Reply:
yes.. It was not to debug fault situation.
reverted it.